radar_parlamentar/importadores/tests/mocks_cdep.py

```diff
@@ -27,6 +27,9 @@
 
 def parse_arquivo_xml(nome_arquivo):
     """retorna etree"""
+
```

Comment:
Sem prints no meio do código pessoal....

Eventualmente podemos até usar para debug, mas, melhor que isso, seria usar logger.debug.
E se não for mais necessário, é só retirar ;)